benchmarks/Lua/bounce.lua

```diff
@@ -0,0 +1,97 @@
+-- This code is derived from the SOM benchmarks, see AUTHORS.md file.
+--
+-- Copyright (c) 2016 Francois Perrad <francois.perrad@gadz.org>
+--
+-- Permission is hereby granted, free of charge, to any person obtaining a copy
+-- of this software and associated documentation files (the 'Software'), to deal
+-- in the Software without restriction, including without limitation the rights
+-- to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+-- copies of the Software, and to permit persons to whom the Software is
+-- furnished to do so, subject to the following conditions:
+--
+-- The above copyright notice and this permission notice shall be included in
+-- all copies or substantial portions of the Software.
+--
+-- THE SOFTWARE IS PROVIDED 'AS IS', WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+-- IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+-- FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+-- AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+-- LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+-- OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+-- THE SOFTWARE.
+
+local Random = require'som'.Random
+
+local Ball = {_CLASS = 'Ball'} do
```

Comment:
I use 2 templates :
 - one for single object
```lua
local foo = {} do

function foo:bar ()
    -- do something
end

end -- object foo
```
- another for class
```lua
local Foo = {_CLASS = 'Foo'} do

function Foo.new ()
    local obj = {}
    return setmetatable(obj, {__index = Foo})
end

function Foo:bar ()
    -- do something
end

end -- class Foo
```
`_CLASS` is not mandatory, but useful when debugging because you could `print(obj._CLASS)`.
Using `_CLASS`  with a single object seems weird.
The `do` in first line and `end` in last line allow to create a scope for local variables.